tools/cldr-code/src/main/java/org/unicode/cldr/util/PathDescription.java

```diff
@@ -93,22 +106,32 @@ public enum Status {
 
     public String getRawDescription(String path, Object context) {
         status.clear();
-        return pathHandling.get(path, context, pathArguments);
+        return pathHandling.get(path, context, pathArguments).getSecond();
     }
 
     public String getRawDescription(
             String path, Object context, Output<Finder> matcherFound, Set<String> failures) {
         status.clear();
-        return pathHandling.get(path, context, pathArguments, matcherFound, failures);
+        return pathHandling.get(path, context, pathArguments, matcherFound, failures).getSecond();
     }
 
     public String getDescription(String path, String value, Object context) {
         status.clear();
 
-        String description = pathHandling.get(path, context, pathArguments);
-        if (description == null) {
+        final Pair<String, String> entry = pathHandling.get(path, context, pathArguments);
+        String description;
+        String markdown;
+        if (entry == null) {
+            markdown = MISSING_DESCRIPTION;
+            description = null;
+        } else {
+            description = entry.getFirst();
+            markdown = entry.getSecond();
+        }
+
+        if (description == null || description.isEmpty()) {
             description = MISSING_DESCRIPTION;
-        } else if ("SKIP".equals(description)) {
```

Comment:
If it starts with SKIP, but has other characters, it is an error; so best to raise an exception.

Reply:
the description here is more rich, it's for you to name (or in the future group) these entries.

as below, where First Day of the Week is a comment.

    ### First Day of the Week

    - `^//ldml/localeDisplayNames/types/type\[@key="fw"]\[@type="([^"]*)"]`

    The name of “first day of the week is {1}”. For more information, please see [Key Names].